components/header/index.css

```diff
@@ -0,0 +1,16 @@
+.header-area-user {
+  justify-content: flex-end;
+}
+
+.header-item.header-area-navigation {
```

Comment:
Looks weird in comparison to selector above. Write either with `.header-item` or without it but be consistent. Unless you cannot, write a comment please.

I would prefer without because it's clearer unless you want to increase specifity to override declarations without `!important`.

Just want to clarify 🧐

Reply:
I wanted to avoid !important